tests/test_custom_transformer.py

```diff
@@ -13,6 +15,31 @@
 # pylint: disable=missing-function-docstring, missing-class-docstring
 
 
+def test_email_transformer_in_pipeline(X_strings) -> None:
+    pipeline = make_pipeline(EmailTransformer())
+    result = pipeline.transform(X_strings[["email"]])
+    expected = pd.DataFrame(
+        {
+            "email": {
+                0: "test",
+                1: "test123",
+                2: "test_123$$",
+                3: "test_test",
+                4: "ttt",
+                5: "test_test_test",
+            },
+            "email_domain": {0: "com", 1: "com", 2: "com", 3: "com", 4: "com", 5: None},
```

Comment:
The following suggestion is to be made, if this change was made: [https://github.com/chrislemke/feature-reviser/pull/1/files#r977449460](url).
```suggestion
            "email_domain": {0: "test", 1: "test", 2: "test", 3: "test", 4: "test", 5: None},
```